### PR TITLE
674 convert context folder files to typescript

### PR DESCRIPTION
**Pull request by deekshithepns**
Converted files in context folder to typescript

### CODE REVIEW

File: src/contexts/VideoCallContext.tsx [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+  const myVideo = useRef();
+  const userVideo = useRef();
+  const connectionRef = useRef();
+
+  const initializeStream = async (address: string): Promise<void> => {
+    try {
+      const stream = await navigator.mediaDevices.getUserMedia({ video: true, audio: true });
+      setStream(stream);
+      myVideo.current.srcObject = stream;
```
Comment (mishramonalisha76) [inline src/contexts/VideoCallContext.tsx:48]: fix the error 'myVideo.current' is possibly 'undefined' in this line, also fix this in the entire file, there are quite a few places with typescript  error.

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]

File: src/contexts/VideoCallContext.tsx [hunk 2]
```diff
@@ -0,0 +1,243 @@
+// React + Web3 Essentials
+import React, { createContext, useEffect, useRef, useState } from 'react';
+
+// External Packages
+import Peer from 'simple-peer';
+import { io } from 'socket.io-client';
+
+// Internal Configs
+import { appConfig } from 'config';
```
[the remark this answers is not in the document]
Comment (deekshithepns) [reply src/contexts/VideoCallContext.tsx:9]: ok, I wil fix this

File: src/contexts/VideoCallContext.tsx (hunk 1 above)
Comment (deekshithepns) [reply src/contexts/VideoCallContext.tsx:48]: ok...I will look into it

Comment (deekshithepns) [reply src/contexts/VideoCallContext.tsx:48]: Pushed changes...

**Review by mishramonalisha76 — DISMISSED** [review]
LGTM

**Comment by zaryab2000** [comment]
@mishramonalisha76

**Comment by mishramonalisha76** [comment]
have some console errors in this to fix

**Review by mishramonalisha76 — DISMISSED** [review]

**Pull request merged**